### PR TITLE
Automated endpoint response actions on rules (kill/suspend process)

### PR DESCRIPTION
**Pull request by joepeeples**
Contributes to https://github.com/elastic/security-docs/issues/4874.

### Previews
* [Automated response actions](https://security-docs_bk_4890.docs-preview.app.elstc.co/guide/en/security/master/automated-response-actions.html) – New page with overall instructions on adding automated response actions to rules.
* [Create a detection rule | Set up response actions (optional)](https://security-docs_bk_4890.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#rule-response-action) — Revise section to point to new page above, and a bit o' reformatting.

### Related docs PR

https://github.com/elastic/security-docs/pull/4885 makes some organizational changes to rearrange the response action pages into a top-level section. This will affect the current PR since it also creates a new page in this section. Expect some light conflicts when either of these is merged, but should be easily resolvable.

### Serverless docs
No serverless docs PR open yet. I'll wait until the content is finalized in this classic/ESS PR, then open a serverless docs PR with the same content changes.

### CODE REVIEW

**Review by tomsonpl — DISMISSED** [review]
LGTM 👍  Thanks!

**Review by ashokaditya — APPROVED** [review]

**Review by caitlinbetz — DISMISSED** [review]

File: docs/detections/rules-ui-create.asciidoc [hunk 1]
```diff
@@ -681,11 +681,11 @@ Example using the mustache "current element" notation `{{.}}` to output all the
...
 === Set up response actions (optional)
-Use Response Actions to set up additional functionality that will run whenever a rule executes.
+Use Response Actions to set up additional functionality that will run whenever a rule executes:
 
-The Osquery Response Action allows you to include live Osquery queries with a custom query rule. When an alert is generated, Osquery automatically collects data on the system related to the alert. Refer to <<osquery-response-action>> to learn more.
+* **Osquery**: Include live Osquery queries with a custom query rule. When an alert is generated, Osquery automatically collects data on the system related to the alert. Refer to <<osquery-response-action>> to learn more.
 
-The Endpoint Security response action allows you to automatically run response actions on an endpoint when rule conditions are met. Refer to <<isolate-a-host>> to learn more.
+* **{elastic-defend}**: Automatically run response actions on an endpoint when rule conditions are met. For example, you can automatically isolate a host or terminate a process when specific activity or events are detected on the host. Refer to <<automated-response-actions>> to learn more.
```
[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/detections/rules-ui-create.asciidoc:688]: Good catch; I like plural too:
```suggestion
* **{elastic-defend}**: Automatically run response actions on an endpoint when rule conditions are met. For example, you can automatically isolate a host or terminate a process when specific activities or events are detected on the host. Refer to <<automated-response-actions>> to learn more.
```

File: docs/detections/rules-ui-create.asciidoc [hunk 2]
```diff
@@ -681,11 +681,11 @@ Example using the mustache "current element" notation `{{.}}` to output all the
 [float]
 [[rule-response-action]]
 === Set up response actions (optional)
-Use Response Actions to set up additional functionality that will run whenever a rule executes.
+Use Response Actions to set up additional functionality that will run whenever a rule executes:
 
-The Osquery Response Action allows you to include live Osquery queries with a custom query rule. When an alert is generated, Osquery automatically collects data on the system related to the alert. Refer to <<osquery-response-action>> to learn more.
+* **Osquery**: Include live Osquery queries with a custom query rule. When an alert is generated, Osquery automatically collects data on the system related to the alert. Refer to <<osquery-response-action>> to learn more.
```
Comment (benironside) [inline docs/detections/rules-ui-create.asciidoc:686]: I noticed that the link for <<osquery-response-action>> appears in italics in the preview but the on the following line 687 the <<automated-response-actions>> link is not italicized. Curious. Any idea why?

**Review by benironside — APPROVED** [review]
LGTM! Just left one very minor question

File: docs/detections/rules-ui-create.asciidoc (hunk 2 above)
Comment (joepeeples) [reply docs/detections/rules-ui-create.asciidoc:686]: @benironside That's honestly one of the biggest mysteries of AsciiDoc. I've noticed it several times and haven't ever figured out why it italicizes sometimes, and doesn't other times. It doesn't seem like something we can control, or at least not without a deep dive into our CSS stylesheet or other inner workings.

**Pull request merged**